PostProcessing/Runtime/Components/DepthOfFieldComponent.cs

```diff
@@ -105,7 +105,8 @@ public void Prepare(RenderTexture source, Material uberMaterial, bool antialiasC
             material.SetFloat(Uniforms._RcpAspect, 1f / aspect);
 
             // CoC calculation pass
-            var rtCoC = context.renderTextureFactory.Get(context.width, context.height, 0, cocFormat);
+			// HACK by hdh, add RenderTextureReadWrite.Linear setting. in DX9 shader model 3 Linear rendering, without this will have bug
```

Comment:
Can you remove this comment? Your name will still appear in the commit log, no need to put it in a comment :)